backend/app/models/option.rb

```diff
@@ -12,6 +12,12 @@ class Option < ApplicationRecord
 
   delegate :single_choice?, to: :question, allow_nil: true
 
+  scope :by_user, lambda { |user|
+    return where(user_id: user).includes([:answers]) unless user.admin?
```

Comment:
```suggestion
    return where(user: user).includes([:answers]) unless user.admin?
```
can you check if this work?

Reply:
Done!